internal-ui/src/pages/UserMessagesPage.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+    Breadcrumb,
+    BreadcrumbItem, Button,
+    PageSection,
+} from "@patternfly/react-core";
+import {Link} from "react-router-dom";
+
+import {MessageUser, Session} from "../Types.ts";
+import {useMessages} from "../services/messages.ts";
+import {LoadingPageSection} from "../components/LoadingPageSection.tsx";
+import { UserMessages } from "../components/UserMessages.tsx";
+
+
+export const UserMessagesPage = () => {
+
+    const messagesQuery = useMessages();
```

Comment:
It is worth to add a param here (or options variable) to be able to set the message type.
I see that you are using `user`.

It's currently sending these https://github.com/RedHatInsights/astro-virtual-assistant/blob/main/internal-ui/src/services/messages.ts#L5-L9 (which is that could be the default if this new param is unset)

Reply:
To see how many times the bot says something could be useful, makes sense :thinking: